npmrc

```diff
@@ -0,0 +1 @@
+registry=https://cli-npm.heroku.com
```

Comment:
Hmm yeah I guess cloudfront would need a SAN cert with both cli-npm.heroku.com and npm.heroku.com. I setup the DNS which is the easy part at any rate, but it won't work for ssl requests until the cert is updated.